src/views/name.njk

```diff
@@ -172,17 +173,33 @@
 
 <script>
   document.addEventListener('DOMContentLoaded', function() {
+    function validateNameLength() {
+      const nameLength = document.getElementById('partnership_name').value.length;
```

Comment:
could be replaced by:
nameField.value.length

Reply:
Done and retested.